config.lua

```diff
@@ -1,3 +1,42 @@
+VehicleClasses = {
+    Compacts = 0,
+    Sedans = 1,
+    SUVs = 2,
+    Coupes = 3,
+    Muscle = 4,
+    Sports_Classics = 5,
+    Sports = 6,
+    Super = 7,
+    Motorcycles = 8,
+    Off_road = 9,
+    Industrial = 10,
+    Utility = 11,
+    Vans = 12,
+    Cycles = 13,
+    Boats = 14,
+    Helicopters = 15,
+    Planes = 16,
+    Service = 17,
+    Emergency = 18,
+    Military = 19,
+    Commercial = 20,
+    Trains = 21,
+    Open_Wheel = 22,
```

Comment:
Should we use ALL_CAPS for enum keys? Up to you.

Reply:
Technically, we should

Reply:
Done